features/finder_frontend.feature

```diff
@@ -53,6 +53,58 @@ Feature: Finder Frontend
     Then I should see filtered documents
 
   @high
-  Scenario: Check malicious code does not execute
-    When I visit "/government/organisations/hm-revenue-customs/contact?keywords=<script>alert(document.cookie);</script>"
-    Then I should see "&lt;script&gt;alert(document.cookie)"
+  Scenario Outline: Check malicious code does not execute
+    When I visit the "<finder>" finder with keywords <keyword>
+    Then There should be no alert
+    And I fill in the keyword field with <keyword>
+    Then There should be no alert
+    And I should see the string <keyword>
```

Comment:
I suppose JS needs to be enabled for this test?

Reply:
I assume all these tests run with JS enabled

Reply:
Do you need the `@javascript` flag as in `/features/finders.feature`?

Reply:
This appears not to be necessary.

Reply:
Will filling in the keyword field have an effect, given that queries are cached when JS is enabled?

Reply:
good point. I added a reload so the cache should be emptied